crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -725,6 +746,17 @@ type Object implements ObjectOwner {
 	The domain that a user address has explicitly configured as their default domain
 	"""
 	defaultNameServiceName: String
+	"""
+	A field on an object that can be added or removed on-the-fly,
+	only affects gas when accessed, and can store heterogenous values.
+	"""
+	dynamicField(dynamicFieldName: DynamicFieldName!): DynamicField
+	"""
+	An object stored as a dynamic field on the parent object.
+	This object is also accessible directly via its address.
```

Comment:
```suggestion
	This object is also accessible off-chain via its address.
```

Clarification here -- this property is only true off-chain.

Reply:
i see, thanks!